media-libs/vidstab/vidstab-9999.ebuild

```diff
@@ -21,6 +21,15 @@ fi
 LICENSE="GPL-2+"
 SLOT="0"
 IUSE="openmp cpu_flags_x86_sse2"
+
+pkg_pretend() {
+	[[ ${MERGE_TYPE} != binary ]] && use openmp && tc-check-openmp
```

Comment:
I am personally not a friend of long `&&` chains. `pgk_*` functions are often split into two halves, one that has to run unconditionally, and one that only has to run if it's a non-binary merge. So I apply the following pattern

```
pgk_pretend() {
     <unconditinal phase>
     [[ ${MERGE_TYPE} == binary ]] && return
    <only run if non-binary merge>
}
```

which is more readable and cleaner wrt to future code additions. But YMMV.

Reply:
that's fine, but 100 ebuilds or so are using exactly this pattern already. If you change all of them (taking care not to kill any other lines), I'm fine with that too. Ultimately, My preferred solution would be to actually move the `MERGE_TYPE` check into `tc-check-openmp` and then just go back to vanilla `use openmp && tc-check-openmp`.